From 1e042492b0911db51eea68671550d19bca5c4e70 Mon Sep 17 00:00:00 2001
From: Gent8 <deboordermounir@gmail.com>
Date: Sun, 19 Oct 2025 13:39:02 +0200
Subject: [PATCH 03/24] Update GitHub Actions workflow to reflect rebranding:
 change MIHON_VERSION to TAISON_VERSION and update artifact naming convention
 accordingly.

---
 .github/workflows/build.yml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/.github/workflows/build.yml b/.github/workflows/build.yml
index 0af420ac4..71eb4beb5 100644
--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -23,10 +23,10 @@ jobs:
         distribution: temurin
     - name: Assemble APK
       run: ./gradlew --no-daemon 'assembleRelease'
-    - name: Set MIHON_VERSION
-      run: echo "MIHON_VERSION=v$(grep --only-matching --perl-regexp 'versionName\s=\s\"\K([0-9\.]+)' './app/build.gradle.kts')+$(git rev-parse --short 'HEAD')" >> "${GITHUB_ENV}"
+    - name: Set TAISON_VERSION
+      run: echo "TAISON_VERSION=v$(grep --only-matching --perl-regexp 'versionName\s=\s\"\K([0-9\.]+)' './app/build.gradle.kts')+$(git rev-parse --short 'HEAD')" >> "${GITHUB_ENV}"
     - name: Upload APK
       uses: actions/upload-artifact@main
       with:
-        name: "kahon_${{ env.MIHON_VERSION }}-github-release"
+        name: "taison_${{ env.TAISON_VERSION }}-github-release"
         path: ./app/build/outputs/apk/release/app-release-unsigned.apk
-- 
2.39.5 (Apple Git-154)

